code/datums/spells/wizard.dm

```diff
@@ -147,6 +147,20 @@
 
 	action_icon_state = "spell_forcewall"
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop
+	name = "Stop Time"
+	desc = "This spell stops time for everyone except for you, allowing you to move freely while your enemies and even projectiles are frozen."
+	charge_max = 500
+	clothes_req = 1
+	invocation = "TOKI WO TOMARE"

```

Comment:
I'm honestly surprised this wasn't a "Stop. Hammertime." joke instead of what is presumably an anime reference.